src/Compilers/Server/VBCSCompilerTests/ServerUtil.cs

```diff
@@ -23,6 +23,8 @@ namespace Microsoft.CodeAnalysis.CompilerServer.UnitTests
 {
     internal static class ProtocolUtil
     {
+        internal const string DefaultWorkingDirectory = @"c:\code";
```

Comment:
>internal [](start = 8, length = 8)

It seems strange to see an invalid path used as a fallback value. Could we remove this constant and use `Path.GetTempPath()` (in the one use) below instead?

Reply:
Reworked this a bit to avoid the fake paths. In some ways this pattern would be ideal if I could guarantee that was an invalid directory but cannot so went with a `TempRoot` solution.